### PR TITLE
Fixing unit test

### PR DESCRIPTION
**Pull request by dwybourn**

### CODE REVIEW

**Comment by csscottlogic** [comment]
I think you need an afterEach in the test file as well? I was expecting something more along these lines:
https://docs.angularjs.org/api/ngMock/service/$httpBackend
I.e. we flush the backend and verify expectations have been met?

**Comment by dwybourn** [comment]
Adding the flush call gives the following error “Error: No pending request to flush”

File: tests/unit/speakerService.jasmine.js [hunk 1]
```diff
@@ -1,15 +1,45 @@
...
+        it('should perform http GET on the given url', function(){
+            $httpBackend.expectGET(backend.url + ":" + backend.port + "/speakers").respond(speakers);
+            service.getSpeakers();
+            $httpBackend.flush();
+        });
+    });
+
+    describe('getSpeakers returns speakers', function() {
+        it('should return the speakers returned from the http request', function() {
+            $httpBackend.whenGET(backend.url + ":" + backend.port + "/speakers").respond(speakers);
```
Comment (lclarksl) [inline tests/unit/speakerService.jasmine.js]: Could have a variable for backend.url + ":" + backend.port, else :shipit:

Comment (dwybourn) [inline tests/unit/speakerService.jasmine.js]: Added variable.

File: tests/unit/speakerService.jasmine.js [hunk 2]
```diff
@@ -1,15 +1,46 @@
...
+    var speakers = [
+        {Name: "Joe Bloggs"}
+    ];
+
     beforeEach(angular.mock.module('BristechSRM'));
 
-    var speakerService;
+    beforeEach(function() {
+        angular.mock.module('BristechSRM', function($provide){
+        $provide.constant('backend', backend);
```
Comment (csscottlogic) [inline tests/unit/speakerService.jasmine.js]: This line should be indented?

**Comment by csscottlogic** [comment]
:shipit:

**Comment by lclarksl** [comment]
:shipit:

**Pull request merged**